CreateSymLinkFile.ps1

```diff
@@ -1,12 +1,14 @@
-Add-Type -AssemblyName PresentationFramework
 # Function to check for elevated privileges
 function Test-Admin {
     $currentUser = New-Object Security.Principal.WindowsPrincipal([Security.Principal.WindowsIdentity]::GetCurrent())
     return $currentUser.IsInRole([Security.Principal.WindowsBuiltInRole]::Administrator)
 }
 
 function Confirm-Overwrite {
-    param ($linkPath)
+    param (
+        $linkPath
+    )
```

Comment:
very unnecessary change for a single parameter. would have been fine if there were more than one. but this way it's not bringing anything except making the file longer.